test/test_linalg.py

```diff
@@ -2377,14 +2380,17 @@ def run_test_singular_input(batch_dim, n):
     def test_pinv(self, device, dtype):
         from torch.testing._internal.common_utils import random_hermitian_pd_matrix
 
+        def _(tensor):
```

Comment:
Instead of this function, can you just do np_A and np_A_pinv below? That would prevent excessive copying if they're on CUDA, too.

Reply:
fixed

Reply:
fixed